app/app.test.js

```diff
@@ -45,7 +45,48 @@ describe("app", () => {
                 votes: expect.any(Number),
               })
             );
-            console.log;
+          });
+      });
+      test.only('status: 200 - responds with an article object that has a proper of "comment_count" that shows the correct number of comments for that individual article', () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then(
+            ({
+              body: {
+                article: { comment_count },
+              },
+            }) => {
+              expect(comment_count).toBe(11);
+            }
+          )
+          .then(() => {
+            return request(app)
+              .get("/api/articles/4")
+              .expect(200)
+              .then(
```

Comment:
this behaviour is checking that a comment count of 0 doesn't error out, which is slightly different to cases where there are comments, so I'd say wrap this in it's own test block